### ISSUE TITLE
Cannot add users or new users to groups or collections

### ISSUE THREAD

**Issue by raaswol** [peerlibrary/peerlibrary#706]
In the popout search bar that comes up when adding members to groups, sharing a pdf with members or sharing collections with members, returns Not Found

groups currently = 1 member, the admin
collections and pdfs that are private cannot be shared

**Comment by sitek** [comment]
I am also having this problem with both groups and collections. Is there an ETA for when this could be back up?

**Comment by mitar** [comment]
Can you describe a bit more the problem? So you do not find any members when you want to add them to the group?

**Comment by sitek** [comment]
I created a group ("Speech Communication Spring 2015"), and when I type in users to invite to the group, it quickly displays "Loading" and then "No people found" and then nothing beyond what I've typed.

Similarly, I created a collection, and the same thing happens when trying to invite users or add my above group - every time I type a letter, a popdown appears, quickly switching between "Loading" and "No people found" and then disappearing without making any suggestions for existing groups or users. 

Lastly, entering a full email, username, or group and hitting enter has no response.

Here's an image, though it's not incredibly helpful:

![screenshot 2015-02-03 15 44 48](https://cloud.githubusercontent.com/assets/2488038/6029013/b007ba4a-abbb-11e4-905e-f8b1ea0a0510.png)
